sphinx_gallery/gen_gallery.py

```diff
@@ -317,6 +319,17 @@ def call_memory(func):
                               % (css, _KNOWN_CSS))
         if gallery_conf['app'] is not None:  # can be None in testing
             gallery_conf['app'].add_css_file(css + '.css')
+    if gallery_conf['thebelab'] is not None:
+        gallery_conf['thebelab']['selector'] = \
```

Comment:
here we should probably use `get` methods, and/or explicitly check for some keys and raise helpful errors if they are required for thebelab to work